src/Microsoft.SymbolStore/KeyGenerators/KeyGenerator.cs

```diff
@@ -39,6 +41,11 @@ public enum KeyTypeFlags
         /// the Portable and Windows PDBs are available on the symbol server.
         /// </summary>
         ForceWindowsPdbs = 0x08,
+
+        /// <summary>
+        /// Generate keys for the host program.
```

Comment:
Can we add a little more description about what the 'host program' relationship is? I assume for a binary 'foo', it is a key with the same buildid renamed to 'apphost'?

Reply:
Done.